src/python/cli_new/lib/cli/config.py

```diff
@@ -137,3 +211,15 @@ def plugins(self):
             return self.data["plugins"]
 
         return []
+
+    def authentication_header(self):
+        """
+        Return the BasicAuth authentication header
+        """
+        if (self.agent_principal() is not None
+                and self.agent_secret() is not None):
```

Comment:
Nit: the indentation seems a bit strange?

Reply:
The linter say it's fine like that. :man_shrugging:  And my vim say, it should be like that. :joy: